go.mod

```diff
@@ -31,8 +31,8 @@ require (
 	google.golang.org/api v0.138.0
 	gopkg.in/yaml.v2 v2.4.0
 	gopkg.in/yaml.v3 v3.0.1
-	k8s.io/api v0.26.4
-	k8s.io/apimachinery v0.26.4
+	k8s.io/api v0.28.0
+	k8s.io/apimachinery v0.28.0
```

Comment:
These can't be done yet, we need this to be resolved first:
https://jira.mongodb.org/browse/CLOUDP-185089
https://github.com/operator-framework/operator-sdk/pull/6514